### ISSUE TITLE
Using stores causes an error in `svelte/valid-compile`

### ISSUE THREAD

**Issue by Serator** [sveltejs/eslint-plugin-svelte#761]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.57.0

### What version of `eslint-plugin-svelte` are you using?

2.39.0

### What did you do?

<details>
<summary>Configuration</summary>

`.eslintrc.cjs`
```
/** @type { import("eslint").Linter.Config } */
module.exports = {
  root: true,
  extends: [
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:svelte/recommended',
    'prettier'
  ],
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],
  parserOptions: {
    sourceType: 'module',
    ecmaVersion: 2020,
    extraFileExtensions: ['.svelte']
  },
  env: {
    browser: true,
    es2017: true,
    node: true
  },
  overrides: [
    {
      files: ['*.svelte'],
      parser: 'svelte-eslint-parser',
      parserOptions: {
        parser: '@typescript-eslint/parser'
      }
    }
  ]
};
```

`svelte.config.js`
```
const config = {
  compilerOptions: {
    runes: true,
  },

  ...
```
</details>

```svelte
<script lang="ts">
  import {page} from '$app/stores'
  $inspect($page)
</script>
```

<img width="421" alt="image" src="https://github.com/sveltejs/eslint-plugin-svelte/assets/625005/d33c451f-9895-453b-b760-50a448e6db59">

### What did you expect to happen?

No errors.

### What actually happened?

When using SvelteKit's inbuilt `$page` store, I get the following error:
```
`$page` is an illegal variable name. To reference a global variable called `$page`, use `globalThis.$page`(global_reference_invalid)  svelte/valid-compile
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-dnsppf?file=src%2Froutes%2F%2Bpage.svelte

**Comment by baseballyama** [comment]
duplicate of https://github.com/sveltejs/eslint-plugin-svelte/issues/652

**Comment by JackNeto** [comment]
This is still an issue for me. 
None of the proposed solutions on #652 worked for me. 
Any suggestions?
